flang/include/flang/Lower/StatementContext.h

```diff
@@ -79,23 +79,26 @@ class StatementContext {
     }
   }
 
-  /// Make cleanup calls. Retain the stack top list for a repeat call.
+  /// Make a cleanup call. Retain the stack top list for a repeat call.
```

Comment:
I think you can keep the original comment since we generate 0 to N cleanup calls.